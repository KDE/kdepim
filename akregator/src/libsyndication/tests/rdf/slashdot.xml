<?xml version="1.0" encoding="ISO-8859-1"?>

<rdf:RDF
 xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
 xmlns="http://purl.org/rss/1.0/"
 xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
 xmlns:taxo="http://purl.org/rss/1.0/modules/taxonomy/"
 xmlns:dc="http://purl.org/dc/elements/1.1/"
 xmlns:syn="http://purl.org/rss/1.0/modules/syndication/"
 xmlns:admin="http://webns.net/mvcb/"
>

<channel rdf:about="http://slashdot.org/">
<title>Slashdot</title>
<link>http://slashdot.org/</link>
<description>News for nerds, stuff that matters</description>
<dc:language>en-us</dc:language>
<dc:rights>Copyright 1997-2004, OSTG - Open Source Technology Group, Inc.  All Rights Reserved.</dc:rights>
<dc:date>2005-04-25T22:00:45+00:00</dc:date>
<dc:publisher>OSTG</dc:publisher>
<dc:creator>pater@slashdot.org</dc:creator>
<dc:subject>Technology</dc:subject>
<syn:updatePeriod>hourly</syn:updatePeriod>
<syn:updateFrequency>1</syn:updateFrequency>
<syn:updateBase>1970-01-01T00:00+00:00</syn:updateBase>
<items>
 <rdf:Seq>
  <rdf:li rdf:resource="http://science.slashdot.org/article.pl?sid=05/04/25/192236&amp;from=rss" />
  <rdf:li rdf:resource="http://it.slashdot.org/article.pl?sid=05/04/25/1710223&amp;from=rss" />
  <rdf:li rdf:resource="http://linux.slashdot.org/article.pl?sid=05/04/25/1321207&amp;from=rss" />
 </rdf:Seq>
</items>
<image rdf:resource="http://images.slashdot.org/topics/topicslashdot.gif" />
<textinput rdf:resource="http://slashdot.org/search.pl" />
</channel>

<image rdf:about="http://images.slashdot.org/topics/topicslashdot.gif">
<title>Slashdot</title>
<url>http://images.slashdot.org/topics/topicslashdot.gif</url>
<link>http://slashdot.org/</link>
</image>

<item rdf:about="http://science.slashdot.org/article.pl?sid=05/04/25/192236&amp;from=rss">
<title>First 500 Terabytes Transmitted via LHCGlobal Grid</title>
<link>http://science.slashdot.org/article.pl?sid=05/04/25/192236&amp;from=rss</link>
<description>neutron_p writes "When the LHC Computer Grid starts operating in 2007, it will be the most data-intensive physics instrument on the planet. Today eight major computing centers successfully completed a challenge to sustain a continuous data flow of 600 megabytes per second on average for 10 days from CERN in Geneva, Switzerland to seven sites in Europe and the US. The total amount of data transmitted during this challenge -- 500 terabytes -- would take about 250 years to download using a typical 512 kilobit per second household broadband connection."</description>
<dc:creator>timothy</dc:creator>
<dc:date>2005-04-25T19:39:00+00:00</dc:date>
<dc:subject>networking</dc:subject>
<slash:department>yeah-but-most-of-it-was-zeroes</slash:department>
<slash:section>science</slash:section>
<slash:hitparade>178,167,113,89,36,18,16</slash:hitparade>
<slash:comments>178</slash:comments>
</item>

<item rdf:about="http://it.slashdot.org/article.pl?sid=05/04/25/1710223&amp;from=rss">
<title>NETI@home Data Analyzed</title>
<link>http://it.slashdot.org/article.pl?sid=05/04/25/1710223&amp;from=rss</link>
<description>An anonymous reader writes "The NETI@home Internet traffic statistics project (featured in Wired and Slashdot previously) has a quick analysis on the malicious traffic they observed. It's a rough world out there." Perhaps not suprising, but still disheartening, the researchers find among other things that a large portion of typical end-user traffic consists of malicious connection attempts.</description>
<dc:creator>timothy</dc:creator>
<dc:date>2005-04-25T18:52:00+00:00</dc:date>
<dc:subject>security</dc:subject>
<slash:department>sharpen-your-knives</slash:department>
<slash:section>it</slash:section>
<slash:hitparade>119,109,79,55,25,17,14</slash:hitparade>
<slash:comments>119</slash:comments>
</item>

<item rdf:about="http://linux.slashdot.org/article.pl?sid=05/04/25/1321207&amp;from=rss">
<title>The Truth About Linux and Windows</title>
<link>http://linux.slashdot.org/article.pl?sid=05/04/25/1321207&amp;from=rss</link>
<description>petrus4 writes "Groklaw has an update on the Laura DiDio saga. Apparently, her complaints about "Linux extremists" notwithstanding, cooler heads than the usual suspects are asking questions about her research. A very interesting read, and one which will hopefully encourage corporate readers to regard the Yankee Group's findings with the requisite metric ton of salt in the future."</description>
<dc:creator>Hemos</dc:creator>
<dc:date>2005-04-25T18:10:00+00:00</dc:date>
<dc:subject>linuxbiz</dc:subject>
<slash:department>getting-under-the-covers</slash:department>
<slash:section>linux</slash:section>
<slash:hitparade>395,380,278,186,59,32,22</slash:hitparade>
<slash:comments>395</slash:comments>
</item>

<textinput rdf:about="http://slashdot.org/search.pl">
<title>Search Slashdot</title>
<description>Search Slashdot stories</description>
<name>query</name>
<link>http://slashdot.org/search.pl</link>
</textinput>

</rdf:RDF>
