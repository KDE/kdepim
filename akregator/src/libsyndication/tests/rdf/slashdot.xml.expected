# Feed begin ######################
title: #Slashdot#
link: #http://slashdot.org/#
description: #News for nerds, stuff that matters#
copyright: #Copyright 1997-2004, OSTG - Open Source Technology Group, Inc.  All Rights Reserved.#
language: #en-us#
# Person begin ####################
email: #pater@slashdot.org#
# Person end ######################
# Image begin #####################
url: #http://images.slashdot.org/topics/topicslashdot.gif#
title: #Slashdot#
link: #http://slashdot.org/#
# Image end #######################
# Item begin ######################
id: #http://science.slashdot.org/article.pl?sid=05/04/25/192236&from=rss#
title: #First 500 Terabytes Transmitted via LHCGlobal Grid#
link: #http://science.slashdot.org/article.pl?sid=05/04/25/192236&from=rss#
description: #neutron_p writes &quot;When the LHC Computer Grid starts operating in 2007, it will be the most data-intensive physics instrument on the planet. Today eight major computing centers successfully completed a challenge to sustain a continuous data flow of 600 megabytes per second on average for 10 days from CERN in Geneva, Switzerland to seven sites in Europe and the US. The total amount of data transmitted during this challenge -- 500 terabytes -- would take about 250 years to download using a typical 512 kilobit per second household broadband connection.&quot;#
datePublished: #Mon Apr 25 21:39:00 2005#
dateUpdated: #Mon Apr 25 21:39:00 2005#
# Person begin ####################
name: #timothy#
# Person end ######################
commentsCount: #178#
# Item end ########################
# Item begin ######################
id: #http://it.slashdot.org/article.pl?sid=05/04/25/1710223&from=rss#
title: #NETI@home Data Analyzed#
link: #http://it.slashdot.org/article.pl?sid=05/04/25/1710223&from=rss#
description: #An anonymous reader writes &quot;The NETI@home Internet traffic statistics project (featured in Wired and Slashdot previously) has a quick analysis on the malicious traffic they observed. It's a rough world out there.&quot; Perhaps not suprising, but still disheartening, the researchers find among other things that a large portion of typical end-user traffic consists of malicious connection attempts.#
datePublished: #Mon Apr 25 20:52:00 2005#
dateUpdated: #Mon Apr 25 20:52:00 2005#
# Person begin ####################
name: #timothy#
# Person end ######################
commentsCount: #119#
# Item end ########################
# Item begin ######################
id: #http://linux.slashdot.org/article.pl?sid=05/04/25/1321207&from=rss#
title: #The Truth About Linux and Windows#
link: #http://linux.slashdot.org/article.pl?sid=05/04/25/1321207&from=rss#
description: #petrus4 writes &quot;Groklaw has an update on the Laura DiDio saga. Apparently, her complaints about &quot;Linux extremists&quot; notwithstanding, cooler heads than the usual suspects are asking questions about her research. A very interesting read, and one which will hopefully encourage corporate readers to regard the Yankee Group's findings with the requisite metric ton of salt in the future.&quot;#
datePublished: #Mon Apr 25 20:10:00 2005#
dateUpdated: #Mon Apr 25 20:10:00 2005#
# Person begin ####################
name: #Hemos#
# Person end ######################
commentsCount: #395#
# Item end ########################
# Feed end ########################

